modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,155 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="proxy-certificates_{context}"]
+= Proxy certificates
+
+[discrete]
+== Purpose
+
```

Comment:
It might be helpful to provide a higher-level purpose of the proxy certs before diving right into the details of the `trustedCA` field. An intro similar to the following should be considered:

```
Proxy certificates allow users to specify one or more custom certificate authority (CA) certificates used by platform components when making egress connections.